SDWebImage/UIImage+GIF.m

```diff
@@ -87,7 +87,9 @@ + (UIImage *)sd_animatedGIFNamed:(NSString *)name {
     CGFloat scale = [UIScreen mainScreen].scale;
 
     if (scale > 1.0f) {
-        NSString *retinaPath = [[NSBundle mainBundle] pathForResource:[name stringByAppendingString:@"@2x"] ofType:@"gif"];
+        int imgScale = [NSNumber numberWithFloat:scale].intValue;
```

Comment:
actually, we should be using NSUInteger here `(NSNumber. unsignedIntegerValue)` so that it's safe for both 32bit and 64bit architectures.